ext/standard/tests/file/lstat_stat_variation15.phpt

```diff
@@ -34,10 +34,10 @@ symlink($filename, $linkname);
 echo "*** Testing lstat() on a link after changing its access permission ***\n";
 clearstatcache();
 $old_stat = lstat($linkname);
+sleep(1);
 var_dump( chmod($linkname, 0777) );
 // clear the stat
 clearstatcache();
-sleep(2);
 $new_stat = lstat($linkname);
 // compare self stats
 var_dump( compare_self_stat($old_stat) );
```

Comment:
Should we drop this check, as now they're different?

Reply:
I believe we need to replace

```
var_dump( compare_stats($old_stat, $new_stat, $all_stat_keys, "==") );
// with
$affected_members = array(8, 'atime');
var_dump( compare_stats($old_stat, $new_stat, $affected_members, "<") );
```

below.

Reply:
@nikic Indeed. I just took a look at `lstat_stat_variation16.phpt` and it has a similar situation:

https://github.com/php/php-src/blob/2cfb5250969016d940d856b2ea84ed1a6ce4f6aa/ext/standard/tests/file/lstat_stat_variation16.phpt#L39-L41